### ISSUE TITLE
Is TX,TY,TZ in image.txt formatted as BLH or XYZ when I get GPS BLH information from image's EXIF?

### ISSUE THREAD

**Issue by tangeroo** [colmap/colmap#1071]

**Comment by TroiiF** [comment]
No, the world coord system in colmap is not the same as GPS coord. 
colmap uses Cartesian coordinate system which can be seemed like XYZ system while GPS is some specific BLH system like WGS84, you need to transfer between them.
